lib/deaf_grandma.rb

```diff
@@ -14,13 +14,27 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      if @bye_counter > 0
+        exit(0)
+      end
     end
   end
 
 
   def speak(input)
 
-    #Implement your code here <<<<<<<<<
+    response = "SPEAK UP SONNY!"
+
+    if input.upcase!.nil?
```

Comment:
Hmmm. I wouldn't have thought of checking the return for nil as a way to determine if there was no change.
Reads a little odd to me.

Not a bug or need to be fixed. Just seems strange.